lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -25,8 +26,8 @@ def self.parse(file)
     end
 
     def self.available?
-      `#{puppet_bin} help strings 2>&1`
-      if $?.success?
+      @raw_json, status = capture2("#{puppet_bin} help strings")
+      if status
```

Comment:
Status is a `Process::Status` which always evaluates to true:
```
[4] pry(main)> x, status = Open3.capture2("puppet help strings")
Error: Unable to find application 'ssl'. Unable to activate puppet-6.14.0, because fast_gettext-2.0.1 conflicts with fast_gettext (~> 1.1)
=> ["", #<Process::Status: pid 873396 exit 1>]
[5] pry(main)> status
=> #<Process::Status: pid 873396 exit 1>
[6] pry(main)> puts 'bla' if status
bla
=> nil
```

```suggestion
      if status.success?
```